dns/opnblock/src/opnsense/mvc/app/controllers/OPNsense/opnblock/forms/general.xml

```diff
@@ -0,0 +1,20 @@
+<form>
+    <field>
+        <id>opnblock.general.Enabled</id>
+        <label>Enable OPNblock</label>
+        <type>checkbox</type>
+        <help>Turn off and on OPNblock's service. This will not control your Unbound DNS server instance.</help>
+    </field>
+    <field>
+        <id>opnblock.general.whitelist</id>
+        <label>Whitelist</label>
+		<help>Enter simple whitelist regex entries, such as /(api.solvemedia.com)/ seperated by a space.</help>
+        <type>text</type>
```

Comment:
you should probably prefer a CSVListField here and later, you can use the select_multiple with tokenize style which allows deleting entries via an "x". It also looks better btw.

Reply:
I didn't even know that existed! Wow! I'd love to try that out.

Reply:
Changed "text" with "CSVListField" here. Wasn't sure if that was the correct way to go about it.